### PR TITLE
[DM-2970]: Create ui-c8y--services-as-a-plugin.md

### PR DESCRIPTION
**Pull request by dafkk**

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y--device-management-services-functionality-is-now-available-as-a-plugin.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-2970
+version:
+---
+The functionality for services has been modularized into a plugin within the device management application. By default, users will have access to services within the application. However, if needed, the plugin can be removed, thus disabling the services within the application. This approach offers flexibility for users who may or may not require certain functionalities within the application.
```
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y--device-management-services-functionality-is-now-available-as-a-plugin.md:17]:
```suggestion
The functionality for services has been modularized into a plugin. By default, users will have access to services within the device management application. However, if not needed, the functionality can be removed by deactivating the plugin. This approach offers flexibility for users who may or may not require certain functionalities within the application.
```

File: content/change-logs/device-management/ui-c8y--device-management-services-functionality-is-now-available-as-a-plugin.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-2970
+version:
+---
+The functionality for services has been modularized into a plugin. By default, users will have access to services within the device management application. However, if not needed, the functionality can be removed by deactivating the plugin. This approach offers flexibility for users who may or may not require certain functionalities within the application.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y--device-management-services-functionality-is-now-available-as-a-plugin.md:17]:
```suggestion
The functionality for services has been modularized into a plugin. By default, users will have access to services within the Device Management application. However, if not needed, the functionality can be removed by deactivating the plugin. This approach offers flexibility for users who may or may not require certain functionalities within the application.
```

File: content/change-logs/device-management/ui-c8y--device-management-services-functionality-is-now-available-as-a-plugin.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2024-05-07
+title: Device management services functionality is now available as a plugin.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y--device-management-services-functionality-is-now-available-as-a-plugin.md:3]:
```suggestion
title: Device management services functionality is now available as a plugin
```

**Review by BeateRixen — APPROVED** [review]

**Review by bozhidardedov — APPROVED** [review]

**Comment by BeateRixen** [comment]
@dafkk This file is missing a version and should probably not have a date yet. I already removed the date from the merged file so that is does not show up. What is the version that must be added?

**Comment by dafkk** [comment]
> @dafkk This file is missing a version and should probably not have a date yet. I already removed the date from the merged file so that is does not show up. What is the version that must be added?

It is missing a version because it is not merged to the cd branch, so not released right away and it does not have a version. As far as I understood with the features we merge to develop only we only add date and no version but let me know if I am mistaken.

https://github.softwareag.com/IOTA/cumulocity-ui/pull/4897 - no version available yet.

**Comment by BeateRixen** [comment]
> > @dafkk This file is missing a version and should probably not have a date yet. I already removed the date from the merged file so that is does not show up. What is the version that must be added?
> 
> It is missing a version because it is not merged to the cd branch, so not released right away and it does not have a version. As far as I understood with the features we merge to develop only we only add date and no version but let me know if I am mistaken.
> 
> https://github.softwareag.com/IOTA/cumulocity-ui/pull/4897 - no version available yet.

If you add a date the entry is directly published in the documentation for CD. Please leave the date empty if it is not yet released (as said I already removed it as the ticket showed up on the doc website). The date must be synced with the deployment date on Zone 1 later.

**Pull request merged**